MonoGame.Framework/Platform/Native/GamePlatform.Native.cs

```diff
@@ -11,54 +17,138 @@ partial class GamePlatform
 
 class NativeGamePlatform : GamePlatform
 {
-    public NativeGamePlatform(Game game) : base(game)
+    internal unsafe MGP_Platform* Handle;
+
+    private static unsafe MGG_GraphicsSystem* _system;
+
+    private NativeGameWindow _window;
+
+    public unsafe NativeGamePlatform(Game game) : base(game)
     {
+        Handle = MGP.Platform_Create(out GameRunBehavior behavior);
+        Keyboard.Handle = Handle;
+
+        DefaultRunBehavior = behavior;
+
+        _window = new NativeGameWindow(this, "MainGameWindow");
+
+        Window = _window;
+    }
+
+    internal static unsafe MGG_GraphicsSystem* GraphicsSystem
+    {
+        get
+        {
+            if (_system == null)
+            {
+                Environment.SetEnvironmentVariable("VK_LOADER_DEBUG", "all");
+                Environment.SetEnvironmentVariable("VK_LAYER_PATH", "C:\\VulkanSDK\\1.3.283.0\\Bin");
```

Comment:
Looks very os specific

Reply:
Yeah I already removed it. It was some hacked code when I was trying to fix Vulkan validation.